specification/cognitiveservices/OpenAI.Inference/models/completions.create.cadl

```diff
@@ -138,6 +138,9 @@ model Completions {
     "model"?: string;
     @doc("Array of choices returned containing text completions to prompts sent")
     choices?: Choice[];
+    @added(ServiceApiVersions.v2022_12_01)
```

Comment:
This is untrue, as this field is already available in the preview API version and was not added in 2022-12-01. I would remove the line